indigo_api/tests/xslt_fixtures/act_text-general-input.xml

```diff
@@ -69,911 +60,911 @@
       <p>AND NOW THEREFORE, BE IT ENACTED by the City of Cape Town, as follows:—</p>
     </preamble>
     <body>
-      <chapter id="chapter-1">
+      <chapter eId="chp_1">
         <num>1</num>
         <heading>Definitions and application</heading>
-        <section id="section-1">
+        <section eId="sec_1">
           <num>1.</num>
           <heading>Definitions</heading>
-          <paragraph id="section-1.paragraph-0" refersTo="#term-venue_owner">
+          <hcontainer eId="sec_1__hcontainer_0">
```

Comment:
```suggestion
          <hcontainer eId="sec_1__hcontainer_1">
```

Reply:
Nothing's numbered from 0 anymore, right?

Reply:
This implies Slaw is getting it wrong. Will check.

Reply:
Updated slaw on AKN3 to do correct eId numbers, and updated this test.